api/src/main/resources/application.yml

```diff
@@ -34,6 +34,19 @@ spring:
       enabled: false
     kubernetes:
       enabled: false
+    aws:
+      credentials:
+        access-key: ${csm.platform.s3.accessKeyId}
+        secret-key: ${csm.platform.s3.secretAccessKey}
+      s3:
+        # We don't need/have a region for our local S3 service but the AWS SDK requires one to be configured
+        region: ${csm.platform.s3.region}
```

Comment:
The original/old implementation had a hardcoded `dummy` value here. I don't think we really need to be able to configure that part. But if you want to, should we have the corresponding `region` entry in the API properties. Assuming it is not an error to refer to non existing entries, then this will be empty for now which if I recall doesn't work.

Reply:
I did this in a "last-time minute" change in this PR [#289](https://github.com/Cosmo-Tech/cosmotech-api-common/pull/289)